index.html

```diff
@@ -104,25 +117,51 @@ <h1 style="text-align: center; ">
 community will mirror the information for the popular identifier and setup
 a working redirect to restore service to the rest of the Web.
         </p>
+        <h2 id="new">Creating a new identifier</h2>
```

Comment:
Looks like this should be `h3`?

Reply:
Oh, I just made it even bigger since it's the sub-heading that's surely the most important! Could easily be an h3 though... but then perhaps the others should all be h2s...